config/jobs/kubernetes-sigs/ingress2gateway/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+- bowei
+- thockin
+- aojea
+- robscott
+approvers:
+- bowei
+- thockin
+- robscott
```

Comment:
This does not need to block since apparently we already have the same config for gateway-api, but we should update both to include all Gateway API maintainers ASAP.